### DISCUSSION TITLE
Compatibility with IINA?

### DISCUSSION THREAD

**Discussion by tennyson-mccalla** [pystardust/ani-cli#188] [Q&A] [answered]
I've tried replacing the mpv in player_fn="mpv" with iina to no success. Though I've only tried one episode of one show it succeeds in mpv and fails in IINA. Can anyone help diagnose?

<img width="372" alt="Screen Shot 2021-12-11 at 12 23 40" src="https://user-images.githubusercontent.com/14319110/145685794-a12866df-aba5-48dc-b13c-d1c6310ed098.png">

**Comment by port19x** [comment, marked as answer]
Please open an issue

**Reply by tennyson-mccalla** [reply]
Will do.
